EpiserverRedirects/Configuration/Configuration.cs

```diff
@@ -2,9 +2,9 @@ namespace Forte.EpiserverRedirects.Configuration
 {
     public static class Configuration
     {
-        public static bool PreserveQueryString = false;
-        public static bool AddAutomaticRedirects = true;
-        public static int SystemRedirectRulePriority= 100;
-        public static int DefaultRedirectRulePriority = 100;
+        public const bool PreserveQueryString = false;
```

Comment:
do we really want to change this to const? With const there's no way for end users to alter configuration's values

Reply:
You're right, I forgot it's a lib. Changed back.